.cirrus.yml

```diff
@@ -360,6 +370,9 @@ task:
         CCACHE_MAXSIZE: "400M" # tests two different builds
         SANITIZER_FLAGS: -fsanitize=alignment,undefined
 
+      install_meson_and_rebase_script: |
+        bash src/tools/ci/install_meson_and_rebase.sh ${MESON_REPO} ${MESON_BRANCH} linux
+
```

Comment:
I think it's probably worth just doing this for all tasks, rather than just this one in the matrix.

I'd really like if we could put the invocation into a single place instead of needing to repeat it in many places. But it's not exactly obvious how. Looks like I should have made all tasks in postgres' .cirrus inline the same yaml block, to make it easier to do stuff like this.

I was wondering if we could use https://cirrus-ci.org/guide/programming-tasks/ to make this easier, but I couldn't find a way.

Reply:
Yes, copying same thing on multiple places doesn't seem good but I couldn't find a way too.

Reply:
I wonder, not as part of this PR, whether we should something like
```yaml
task_prep: &task_prep
  # add scripts to be executed at the start of all tasks here

task_finish: &task_finish
  # add scripts to be executed at the end of all tasks here
```

I would like to filter the set of artifacts uploaded in case of failures, which would be a lot easier if we had something like this...

Reply:
Yes, I was thinking if we could ask for a feature request for something like this to cirrus?